react-native-libraries.json

```
@@ -13002,5 +13002,13 @@
     "android": true,
     "web": true,
     "expoGo": true
-  }
+  },
+  {
+    "githubUrl": "https://github.com/jaredh159/tailwind-react-native-classnames",
+    "npmPkg": "twrnc",
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true
+  },
```

Hello @sabakukhyraa, thanks for the addition! 👍  Let's fix the syntax error, so we can validate the entry on the CI:

```suggestion
  }
```

---

Oops, forgot the comma! 😅 Thanks!